pkgs/top-level/all-packages.nix

```diff
@@ -8900,6 +8900,8 @@ with pkgs;
 
   hex = callPackage ../tools/misc/hex { };
 
+  hexbinhex = callPackage ../tools/misc/hexbinhex { };
```

Comment:
Needs to be removed. See comment on djent PR.

Reply:
I have changed it similar to the mentioned PR.